CMakeLists.txt

```diff
@@ -34,9 +34,15 @@ cmake_policy(SET CMP0011 NEW)
 cmake_policy(SET CMP0012 NEW)
 
 if(TEST_CPP)
-    project("Mbed TLS" LANGUAGES C CXX)
+    project("Mbed TLS"
+	    LANGUAGES C CXX
+	    VERSION 3.5.2
```

Comment:
This is going to add maintenance overhead to every release we make in the future.

Would it make sense to put something in `scripts/bump_version.sh`?

Reply:
Ah yes, it does